index.html

```diff
@@ -1,15 +1,31 @@
 <!DOCTYPE html>
-<html lang="en">
+<html>
 <head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Segunda-Tarea</title>
+    <title>Mi Tienda Online</title>
+    <link rel="stylesheet" href="index.css">
 </head>
 <body>
-  <header></header>
-  <main></main>
+    <header>
+        <div class="container">
+            <div class="logo">
+                <img src="image /logo.png">
+            </div>
+            <div class="search-bar">
+                <input type="text" placeholder="¿Qué buscas?">
+                <button><i class="fas fa-search"></i></button>
+            </div>
+            <div class="user-options">
+                <button>Todas</button>
+                <button>Hola</button>
+                <button><i class="fas fa-heart"></i></button>
+                <button><i class="fas fa-shopping-cart"></i></button>
+              
+            </div>
+        </div>
+    </header>
```

Comment:
La tabulación esta rota aqui
```suggestion
  <header>
    <div class="container">
      <div class="logo">
        <img src="image /logo.png">
      </div>
      <div class="search-bar">
        <input type="text" placeholder="¿Qué buscas?">
        <button><i class="fas fa-search"></i></button>
      </div>
      <div class="user-options">
        <button>Todas</button>
        <button>Hola</button>
        <button><i class="fas fa-heart"></i></button>
        <button><i class="fas fa-shopping-cart"></i></button>
      </div>
    </div>
  </header>
```